bundle/kubernetes/manifests/nsx-container-plugin-operator.clusterserviceversion.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Networking, Security
     certified: "True"
-    containerImage: vmware/nsx-container-plugin-operator:3.2.0
+    containerImage: vmware/nsx-container-plugin-operatorf
```

Comment:
Maybe mistype here `vmware/nsx-container-plugin-operatorf`.

Reply:
Hi Jianwei, thanks for correction. Fixed.